index.html

```diff
@@ -220,7 +220,12 @@ <h2 class="about-me-title">About Me</h2>
                 <div class="brief-note-div">
                     <h3 class="brief-note">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
                 </div>
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design

| Your Design(Text not align center, text color does not match ) |Figma Design(Text align center) |
| :---:   | :---: | 
|![Screenshot from 2022-12-13 18-58-58](https://user-images.githubusercontent.com/92303891/207421341-1fb17656-1743-495a-ae45-287856435033.png) | ![Screenshot from 2022-12-13 18-59-59](https://user-images.githubusercontent.com/92303891/207421532-af3b179f-1a5b-4e26-ba6c-9d5c93cd64be.png)|

Reply:
@Owusu-Desmond I was not the one who merged the request. It was one of my partners. I believe it was a mistake. I will also make the necessary adjustment above.